ats/pkb/utils.py

```diff
@@ -9,6 +9,13 @@
     "procedure": nodes.ProcedureNode,
 }
 
+STMT_TYPE_MAP = {
+    "stmt": nodes.StmtNode,
+    "assign": nodes.StmtAssignNode,
+    "while": nodes.StmtWhileNode,
+    "if": nodes.StmtIfNode,
+}
```

Comment:
???

Reply:
dodaliśmy to bo w query evaluate z tego korzystamy

Reply:
No kurwa jego mac. Popatrz na gore pliku